sdk/healthinsights/azure-health-insights-radiologyinsights/CHANGELOG.md

```diff
@@ -1,14 +1,22 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0 (2024-05-03)
 
-### Features Added
+- GA release
 
 ### Breaking Changes
 
-### Bugs Fixed
-
-### Other Changes
+- Client Changes
+    - POST call replaced with PUT
+- Request changes:
+    - Renamed createdDateTime into createdAt
```

Comment:
Add "``" for the variable name. e.g. 
 Renamed `createdDateTime` into `createdAt`.
Similar for below lines.

Reply:
Done